static/app/components/globalSelectionLink.tsx

```diff
@@ -32,49 +32,47 @@ type Props = WithRouterProps & {
  *
  * Falls back to <a> if there is no router present.
  */
-class GlobalSelectionLink extends React.Component<Props> {
-  render() {
-    const {location, to} = this.props;
+function GlobalSelectionLink(props: Props) {
+  const {location, to} = props;
```

Comment:
```suggestion
  const {location, to, children} = props;
```

Reply:
I updated the PR and just let children spread into the element